SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -253,6 +251,7 @@ void CTurbSSTSolver::Postprocessing(CGeometry *geometry, CSolver **solver_contai
     su2double dist = geometry->nodes->GetWall_Distance(iPoint);
 
     su2double VorticityMag = GeometryToolbox::Norm(3, flowNodes->GetVorticity(iPoint));
+    if (VorticityMag == 0) VorticityMag = 1e-6; // safety against division by zero
```

Comment:
Is there a reference for this value?
```suggestion
    VorticityMag = max(VorticityMag, 1e-6); // safety against division by zero
```

Reply:
Nope, background: when I look for nan's with fenv I always (only of course with SST) stumble across this nan. So this more of debugging help for me.

So I just took the mother of all eps ... which for me is 1e-6. [Here](http://www.geosci.sfsu.edu/geosciences/classes/m430/Vorticity/Vorticity.htm) a geo-science resource that states values down to 1e-5. Fun fact: everything near the earth-surface (even you) has a vorticity of ~1e-4[1/s]. 

We could go much lower imo, because I just want to prevent a nan. Code-wise I think we are most likely not changing anything 🤞 because VorticityMag is used in a `min( ..., .../(VorticityMag*...) )` setup... but I guess we were getting nan's ... so latter was actually taken :( But the regression tests are fine.

I'll just make it 1e-12, just to be safe